cli/jobs/pipelines-with-components/image_classification_with_densenet/image_cnn_train/image_classification/training.py

```diff
@@ -618,9 +618,14 @@ def train_loop(
     if is_first_rank:
         ts = str(time.time())
         # logdir = os.path.expanduser('~/tensorboard/{}/logs/'.format(os.environ['DLTS_JOB_ID']) + ts)
-        logdir = os.path.expanduser(
-            "~/tensorboard/{}/logs/".format(os.environ["AZ_BATCH_JOB_ID"]) + ts
-        )
+        if "AML_JOB_ID" in os.environ:
+            logdir = os.path.expanduser(
+                "~/tensorboard/{}/logs/".format(os.environ["AML_JOB_ID"]) + ts
```

Comment:
Is this the same as the RunId? If so, I think a standard variable that works across all computes is AZUREML_RUN_ID. If it's different, then I'd ask why and what is it derived from?

Reply:
Changed to using `AZUREML_RUN_ID` instead.